webapp/components/Group/GroupForm.vue

```diff
@@ -169,18 +222,18 @@ export default {
         name: name || '',
         slug: slug || '',
         groupType: groupType || '',
-        about: about || '',
+        goal: goal || '',
         description: description || '',
         locationName: locationName || '',
         actionRadius: actionRadius || '',
         categoryIds: categories ? categories.map((category) => category.id) : [],
       },
       formSchema: {
         name: { required: true, min: NAME_LENGTH_MIN, max: NAME_LENGTH_MAX },
-        slug: { required: false },
+        slug: { required: false, min: NAME_LENGTH_MIN },
         groupType: { required: true },
-        about: { required: true },
-        description: { required: true },
+        goal: { required: false },
```

Comment:
`goal` is called `about` in the database. So I would named it `about`…

Reply:
is changed